MANIFEST.in

```diff
@@ -0,0 +1,14 @@
+include *.md
+include LICENSE
+include *.bib
+
+recursive-include src/pyradiosky/data *
+
+recursive-include tests *.py
+
+recursive-include docs *
+exclude docs/.DS_Store
+prune docs/_static
+prune docs/_templates
+prune docs/build
+prune docs/__pycache__
```

Comment:
What's the purpose of including the MANIFEST? I thought that using `setuptools_scm` basically made this redundant?

Reply:
Nope. Setuptools_scm doesn't replace the MANIFEST. See details here: https://setuptools.pypa.io/en/latest/userguide/quickstart.html#including-data-files